### ISSUE TITLE
KeeWeb (upstream) needs a new maintainer

### ISSUE THREAD

**Issue by arnowelzel** [jhass/nextcloud-keeweb#204]
As mentioned in https://github.com/keeweb/keeweb/issues/2022 the KeeWeb project which is the upstream for KeeWeb in Nextcloud, needs a new maintainer.  If you are interested in supporting KeeWeb and have the required knowledge and time for it - get in touch with antelle.

For the time being this also means if there are any issues related to the upstream it can not be fixed at the moment.

**Comment by arnowelzel** [comment]
We now reached the point that KeeWeb can not be build any longer.

For the time being there will not be any new release of KeeWeb for Nextcloud, sorry.

Anyone willing and able to fix this, is welcome!

**Comment by MikkCZ** [comment]
Hi @arnowelzel .

Is there a specific issue with KeeWeb ability to be built? I tried locally in my development container, which is based on Fedora 38 with Node.js 18. I was able to build KeeWeb from https://github.com/keeweb/keeweb/tree/e6df216291b9b5322049a11fe1301a2089679483 by running `npm i` followed by `NODE_OPTIONS=--openssl-legacy-provider npx grunt`.

**Comment by MikkCZ** [comment]
I opened #215 to add the `NODE_OPTIONS`. Will that help with a new release, hopefully compatible with NextCloud 26? :wink:

**Comment by arnowelzel** [comment]
> Hi @arnowelzel .
> 
> Is there a specific issue with KeeWeb ability to be built? I tried locally in my development container, which is based on Fedora 38 with Node.js 18. I was able to build KeeWeb from https://github.com/keeweb/keeweb/tree/e6df216291b9b5322049a11fe1301a2089679483 by running `npm i` followed by `NODE_OPTIONS=--openssl-legacy-provider npx grunt`.

No, I forgot to add that the build was possible at last. Anyway the situation will not improve with the unmaintained KeeWeb. I'll try to provide an update for Nextcloud 26 as soon as needed, but if we finally reach the point that it is not possible to use KeeWeb any longer, I don't see any future for this project, sorry!

**Comment by arnowelzel** [comment]
I stand corrected - build is not possible at the moment:

```
Fatal error: error:0308010C:digital envelope routines::unsupported
```

**Comment by arnowelzel** [comment]
Seems https://github.com/jhass/nextcloud-keeweb/pull/215 fixed it indeed - thanks! A new release 0.6.11 for NC 26 was published.

**Comment by ann0see** [comment]
Are there any keepass compatible alternative plugins?

**Comment by arnowelzel** [comment]
For Nextcloud there is no alternative and as far I know, KeeWeb is also the only application to open Keepass databases in a browser.

**Comment by ann0see** [comment]
Oof. Ok. Thank you very much for the information.

**Comment by arnowelzel** [comment]
Just for clarification: a server side implementation would be not secure as the decrypted data must not leave your local computer at any point. So having purely browser based solution is preferrable.

**Comment by ann0see** [comment]
Yes true. On the other hand if you have control over the server and a secure connection it might still be ok-ish for some passwords.

**Comment by luxzg** [comment]
There's more info here regarding progress of v1.19, seems it's doing fine so far: https://github.com/keeweb/keeweb/pull/2168
